ocs_ci/utility/azure_utils.py

```
@@ -544,6 +545,38 @@ def create_noobaa_managed_identity(
+            )
+            try:
+                sa_result = exec_cmd(
+                    f"az storage account show --name {test_storage_account} "
+                    f"--subscription {subscription_id} --query id -o tsv"
+                )
+                sa_resource_id = sa_result.stdout.decode().strip()
+                if sa_resource_id:
+                    for role in roles:
+                        logger.info(
+                            f"Assigning role '{role}' on test storage account "
+                            f"'{test_storage_account}'"
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)